service/vspo-schedule/web/next-i18next.config.js

```diff
@@ -3,9 +3,9 @@ const path = require("path");
 
 module.exports = {
   i18n: {
-    defaultLocale: "ja",
+    defaultLocale: "en",
```

Comment:
I was thinking it might be okay to keep this part in`ja` only 👀 (Since the metadata that was displayed in Japanese so far has become English).

<img width="591" alt="CleanShot 2024-06-29 at 17 42 24@2x" src="https://github.com/sugar-cat7/vspo-portal/assets/69241625/30dbfe78-2d94-4bcb-98ed-b24068e512e5">

On the other hand, each language's page is functioning well, so I will adjust the canonical to ensure proper indexing by the search engine 👍 (separate PR).
<img width="638" alt="CleanShot 2024-06-29 at 17 46 16@2x" src="https://github.com/sugar-cat7/vspo-portal/assets/69241625/fc2b2629-0036-4574-9834-64b17aa19e2a">

Reply:
Sorry for the confusing wording.
Please keep the default as Japanese 🙏 (so that existing users can continue using it as is).